test/hulls.jl

```diff
@@ -71,6 +71,73 @@
       pts = vertices(poly)
       chul = hull(pts, method)
       @test nvertices(chul) < length(pts)
+
+      if method == GrahamScan()
+        # simplifying rectangular hull / triangular
+        points = [P2(i - 1, j - 1) for i in 1:11, j in 1:11] |> vec
```

Comment:
```suggestion
        points = [P2(i - 1, j - 1) for i in 1:11 for j in 1:11]
```
This is a Julia trick to flatten a list comprehension with two or more indices. We just need to add more `for` keywords instead of the comma. Can you please confirm that this is the expected flattening order?